Infrastructure "compose.py --obsolete-fillers option docs" (#46883)
CleverRaven/Cataclysm-DDA: doc/TILESET.md

}
```

> `--obsolete-fillers`: print what fillers were skipped and thus ready to be removed.

Reviewer: ```suggestion
`--obsolete-fillers`: print which fillers were skipped and are thus ready to be removed.
```

- Download `libvips` https://libvips.github.io/libvips/install.html

> - Make sure  directories with `python`, `pip` and `libvips` are in your `PATH` [environment variable](https://en.wikipedia.org/wiki/Environment_variable#Windows)

Reviewer: ```suggestion
- Make sure  directories with `python`, `pip`, and `libvips` are in your `PATH` [environment variable](https://en.wikipedia.org/wiki/Environment_variable#Windows)
```

- Press `Windows key + r` to open the "Run" dialog box

> - run `cmd`, it should open the console

Reviewer: ```suggestion
- run `cmd`; it should open the console
```
